### PR TITLE
Fix SickRage/sickrage-issues/issues/232

### PR DESCRIPTION
**Pull request by fernandog**
https://github.com/SickRage/sickrage-issues/issues/232

@duramato

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickbeard/providers/t411.py [hunk 1]
```diff
@@ -125,8 +125,8 @@ def search(self, search_params, age=0, ep_obj=None):  # pylint: disable=too-many
                                 if not all([title, download_url]):
                                     continue
 
-                                seeders = int(torrent['seeders'])
-                                leechers = int(torrent['leechers'])
+                                seeders = try_int(torrent['seeders'], 1)
+                                leechers = try_int(torrent['leechers'], 0)
```
Comment (miigotu) [inline sickbeard/providers/t411.py]: 0 is already the default.
`try_int(torrent['leechers'])`

File: sickbeard/providers/t411.py [hunk 2]
```diff
@@ -125,8 +125,8 @@ def search(self, search_params, age=0, ep_obj=None):  # pylint: disable=too-many
                                 if not all([title, download_url]):
                                     continue
 
-                                seeders = int(torrent['seeders'])
-                                leechers = int(torrent['leechers'])
+                                seeders = try_int(torrent['seeders'], 1)
```
Comment (miigotu) [inline sickbeard/providers/t411.py]: We dont assume 1 seeder.
`try_int(torrent['seeders'])`

File: sickbeard/providers/t411.py [hunk 3]
```diff
@@ -125,8 +125,8 @@ def search(self, search_params, age=0, ep_obj=None):  # pylint: disable=too-many
...
+                                leechers = try_int(torrent['leechers'], 0)
                                 verified = bool(torrent['isVerified'])
                                 torrent_size = torrent['size']
                                 size = convert_size(torrent_size) or -1
```
Comment (miigotu) [inline sickbeard/providers/t411.py]: Can you move the `size = convert_size(torrent_size) or -1` to just before line 148, no need to convert size if we arent going to keep this torrent.

Same with the `verified =`, that can be just before line 140

**Comment by fernandog** [comment]
@miigotu done

**Pull request merged**